src/storage/src/hummock/store/local_hummock_storage.rs

```diff
@@ -305,7 +321,10 @@ impl LocalStateStore for LocalHummockStorage {
                         )
                         .await?;
                     }
-                    kv_pairs.push((key, StorageValue::new_put(value)));
+                    kv_pairs.push((key, SharedBufferValue::Insert(value)));
+                    if let Some(old_values) = &mut old_values {
+                        old_values.push(Bytes::new());
```

Comment:
Will we have an entry in the old value SST for `KeyOp::Insert`?

Reply:
No. In `SharedBufferBatchIterator`, when `IS_NEW_VALUE` is false, at the end of the `next` call, we will have an extra `advance_until_valid_old_value` to skip the `KeyOp::Insert`